interface/forms/telemed/view.php

```diff
@@ -0,0 +1,378 @@
+
+$dated = new DateTime($encounter_data['encounter_date']);
+$dated = $dated->format('Y-m-d');
+$visit_date = oeFormatShortDate($dated);
+
+$query = "select * from openemr_postcalendar_events where pc_pid=? and pc_eventDate=?";
+$appt = sqlQuery($query, array($pid,$encounter_data['encounter_date']));
+$reason = $appt['pc_hometext'];
+
+$query          = "SELECT * FROM users where id = ?";
+$prov_data      = sqlQuery($query, array($appt['pc_aid']));
+$id = $form_id;
+
+?><!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/html">
+<head>
+    <link rel="stylesheet" href="<?php echo $webroot; ?>/interface/main/messages/css/reminder_style.css?v=<?php echo $v_js_includes; ?>" type="text/css">
+    <link rel="stylesheet"  href="<?php echo $GLOBALS['web_root']; ?>/library/css/bootstrap_navbar.css?v=<?php echo $v_js_includes; ?>" type="text/css">
+
+    <?php Header::setupHeader(''); ?>
+    <?php $GLOBALS['medex_enable']='';
+        if (($GLOBALS['medex_enable'] == '1')) {
+            $logged_in = $MedEx->login('1');
+            if (empty($_REQUEST['nomenu'])) {
+                $MedEx->display->navigation($logged_in);
+            }
+            //ask MedEx to build a room and send room keys based on my plan (JWT or otherwise);
+            $data['pid']            = $pid;
+            $data['providerID']     = $appt['pc_aid'];
+            $data['eid']            = $appt['pc_eid'];
+            $data['token']          = $logged_in['token'];
+            $data['pat_fname']      = $pat_data['fname'];
+            $data['pat_lname']      = $pat_data['lname'];
+            $data['pat_email']      = $pat_data['email'];
+            $data['pat_mobile']     = $pat_data['mobile'];
+            $data['pat_allowemail'] = $pat_data['hipaa_allowemail'];
+            $data['pat_allowsms']   = $pat_data['hipaa_allowsms'];
+    
+            $MedEx->curl->setUrl($MedEx->getUrl('custom/TM_bot'));
+            $MedEx->curl->setData($data);
+            $MedEx->curl->makeRequest();
+            $TM = $MedEx->curl->getResponse();
+        } else {
+            $token = openssl_random_pseudo_bytes(4);
+            $token = bin2hex($token);
+            //Brady: add a global to link to other tele-providers like Doxy.me or Zoom?
+            //For now this links to public servers jitsi-meet  maintainers server.
+            //Room must be locked to be HIPAA compliant.
+            $TM['url_provider'] = 'https://meet.jit.si/'.$token;
+            $TM['url_patient'] = 'https://meet.jit.si/'.$token;
+            $TM['label_url_patient'] = 'https://meet.jit.si/'.$token;
+            $TM['label_url_provider'] = 'Open Jitsi-Meet'; //xlt below
+        }
+        
+        $visit_header = "TeleVisit";
+        if ($logged_in) {
+            $visit_header = "TeleMedEx Visit: secure session";
+        } else {
+            $visit_header = "TeleHealth Visit: Jitsi.org server";
+        }
+        $arrOeUiSettings = array(
+            'heading_title' => xl($visit_header),
```

Comment:
translate the constants above or else they will never get into translation pipeline